### ISSUE TITLE
How to customize the mutation field name for the request body?

### ISSUE THREAD

**Issue by yoonslee** [IBM/openapi-to-graphql#262]
Let me know if you need more clarification, but essentially, I know that I am able to change the query or mutation name via adding an `operationId` value to the OpenAPI spec for various endpoints.

However, is there an equivalent for changing the argument name for a generated mutation?

```
mutation { customizedMutationName(generatedInputName: [JSON in request body]) }
```

Here, I would like to customize the value of `generatedInputName` to just be `body`.

**Comment by Alan-Cha** [comment]
@yoonslee Interesting proposal! This is only for request bodies, yes? I think this will improve generated GraphQL interfaces, making them less verbose. Maybe we should even consider having this option set by default.

Should this be some kind of generic option that can assign the same name to all parameters? Or should it be able to target specific ones? Should the value be `body` or `responseBody` or should the user have control over this? 

Thoughts? 

One thing I don't like about this proposal is that we're starting to mix in HTTP terminology into the GraphQL interface, but I think the improvement outweighs the flaws...

**Comment by yoonslee** [comment]
@Alan-Cha Thanks for the quick response. Super excited about using this library btw.

Yes, this would only apply to mutations' input/argument/field names for the payload a.k.a. `requestBody`.

I think a generic input/argument/field name should apply to all mutations, and after thinking more about it, I want it to be `requestBody`, in line with OpenAPI's usage of `requestBody`.

At least in my use case, I am creating a wrapper for an existing, public REST API whose request body I do not care to label; I only want to know that it IS the request body; the top-level mutation or query name already describes what kind of operation I am performing and the kind of data that I need to pass into the request body. The current, auto-generated input/argument/field names for the request body are camel-cased paths, which I don't care to type every time I want to execute the GraphQL query/mutation.

I agree that mixing in the HTTP terminology here is anti-GraphQL, which is not ideal, but I think the `requestBody` deserves an exception.

I found that changing this line of code in `schema_builder.js` does what I want:
```
// Sanitize the argument name
 const saneName = genericRequestBodyFieldName ? "requestBody" : Oas3Tools.sanitize(def.iotName);
```

**Comment by Alan-Cha** [comment]
I see... yes, I think this is a reasonable change. My only concern is the chance of a name collision. That's why I'm not sure if we should allow the user to input a name. For now, we can just settle on a default name. 

I can make a PR or feel free to file yourself 😄

**Comment by yoonslee** [comment]
Dope, will put it on my to-do list. I rarely do open-source work, so I would love to help out!

**Comment by Alan-Cha** [comment]
Open-source is easy to get started and highly rewarding! No pressure but if you want to give it a go, send us something and we can iterate on it. Don't worry about making it perfect because we're happy to help you through it. Would love to have you on the OtG team! 😄

**Comment by yoonslee** [comment]
@Alan-Cha I finally got around to making the edits needed to implement a new option that sets a generic arg name for mutation request bodies. I tried pushing up my local branch to remote in order to make a PR, but I am not able to.

I apologize if this is obvious, but what are the steps to create a new PR? Should I be forking this repo first?

**Comment by Alan-Cha** [comment]
@yoonslee Wow!!! Awesome! Great to hear!

Yes, you should fork the repo first. If you haven't done so already, you will also need to sign off your commits. You can find more information about contributing [here](https://github.com/IBM/openapi-to-graphql/blob/master/CONTRIBUTING.md). 

If you need any help at all, please ask! I can walk you through all if the steps if need be.

**Comment by yoonslee** [comment]
@Alan-Cha Thanks for your patience! I forked it, signed off on my commits.

Please review PR #269

**Comment by Alan-Cha** [comment]
@yoonslee Just released v2.0.0! Thank you so much for your help @yoonslee!
